planetiler-core/src/test/java/com/onthegomap/planetiler/geo/TileCoordTest.java

```diff
@@ -68,7 +101,8 @@ void testTileSortOrderRespectZ() {
   })
   void testTileProgressOnLevel(int x, int y, int z, double p) {
```

Comment:
and maybe a progress on level for hilbert? that should catch the bug I pointed out

Reply:
Added with z=2 test which fails correctly when bug above is present